src/plugins/comments/__tests__/comments.spec.js

```diff
@@ -864,4 +864,69 @@ describe('Comments', () => {
       expect(plugin.getComment()).toEqual('Bar');
     });
   });
+
+  describe('Destroying the plugin with two instances of Handsontable', () => {
+    it('should create two containers for comments for two HOT instances', () => {
+      const container1 = $('<div id="hot1"></div>').appendTo(spec().$container).handsontable({
+        data: Handsontable.helper.createSpreadsheetData(6, 6),
+        cell: [
+          { row: 1, col: 1, comment: { value: 'Hello world!' } },
+          { row: 1, col: 2, comment: { value: 'Yes!' } }
+        ],
+        rowHeaders: true,
+        colHeaders: true,
+        comments: true,
+        licenseKey: 'non-commercial-and-evaluation'
+      });
+
+      const container2 = $('<div id="hot2"></div>').appendTo(spec().$container).handsontable({
+        data: Handsontable.helper.createSpreadsheetData(6, 6),
+        cell: [{ row: 1, col: 1, comment: { value: 'Hello world!' } }],
+        rowHeaders: true,
+        colHeaders: true,
+        comments: true,
+        licenseKey: 'non-commercial-and-evaluation'
+      });
+
+      const commentContainers = document.querySelectorAll('.htCommentsContainer');
+
+      expect(commentContainers.length).toEqual(2);
+
+      // cleanup HOT instances
+      container1.handsontable('destroy');
+      container2.handsontable('destroy');
```

Comment:
Perhaps just copy the `expect` assertion from above here twice after each destroy to make sure we always have the correct amount of those elements present (same on line 929)

Reply:
Done https://github.com/handsontable/handsontable/pull/7579/commits/137f58458b0b21af2dada6ee077838b61fd0bea4